ads/adspirit.md

```diff
@@ -28,7 +28,7 @@ limitations under the License.
 
 ## Configuration
 
-For semantics of configuration, please see ad network documentation.
+For semantics of configuration, please see ad network documentation at http://help.adspirit.de/help/.
 
```

Comment:
Nit pick: please leverage the markup :-)
`[ad network documentation](http://help.adspirit.de/help/).`